ReactNativeClient/package.json

```diff
@@ -73,6 +73,7 @@
     "react-native-push-notification": "git+https://github.com/laurent22/react-native-push-notification.git",
     "react-native-quick-actions": "^0.3.13",
     "react-native-securerandom": "^1.0.0-rc.0",
+    "react-native-share": "^3.3.3",
```

Comment:
Isn't the Share package of react-native sufficient for this feature? We use it for example in screens/note.js

Reply:
As far as I can see the share package from react-native only supports sharing text.